README.md

```diff
@@ -355,6 +356,75 @@ writable.
 
 Always prefer `ReadonlyArray` over a regular `Array` unless it must be possible to modify the Array.
 
+### React props & TypeScript
+
+#### Why use Interface instead of Type for typing props?
```

Comment:
I think this should be a separate section in #typescript

Reply:
Agree, this is not directly related to React

Reply:
@psimk @ThijsTyZ I did the update.